docs/release-notes/8.13.asciidoc

```diff
@@ -0,0 +1,48 @@
+[[release-notes-header-8.13.0]]
+== 8.13
+
+[discrete]
+[[release-notes-8.13.0]]
+=== 8.13.0
+
+[discrete]
+[[features-8.13.0]]
+==== Features
+
+* Allows you to define an entity's (such as a host's or user's) `Asset criticality`, which can affect risk scores ({kibana-pull}176815[#176815], {kibana-pull}176294[#176294], {kibana-pull}172417[#172417], {kibana-pull}176056[#176056]).
+* Information on the Data Quality dashboard now persists in {elastic-sec} rather than disappearing after each session ({kibana-pull}175673[#175673], {kibana-pull}173185[#173185]).
```

Comment:
```suggestion
* Allows information on the Data Quality dashboard to now persist in {elastic-sec} rather than disappearing after each session ({kibana-pull}175673[#175673], {kibana-pull}173185[#173185]).
```